src/ui/wxWidgets/I18N/Makefile

```diff
@@ -39,7 +39,8 @@ $(POS) : $(POT)
 
 mos/%/LC_MESSAGES/pwsafe.mo : pos/pwsafe_%.po
 	@mkdir -p $(dir $@)
-	@msgfmt --check --output-file=$@ $^
+# do not remove wanted entries:	@msgfmt --check --output-file=$@ $^
```

Comment:
Why keep old code commented out? If the new version works fine, but someone needs to look up the old one for some reason, they need only to look at git blame.

Reply:
Did the change as demanded. Now only a comment about --check option is left.